tests/provider_upgrade_k8s_test.go

```diff
@@ -145,6 +145,6 @@ var _ = Describe("k3s upgrade test", Label("provider", "provider-upgrade-k8s"),
 			out, _ = kubectl(vm, "get pods -A")
 			version, _ := vm.Sudo(getVersionCmd)
 			return version
-		}, 30*time.Minute, 10*time.Second).Should(ContainSubstring("v"), out)
+		}, 30*time.Minute, 10*time.Second).Should(ContainSubstring("v2.3.1-k3sv1.25.11+k3s1"), out)
```

Comment:
maybe this should read the tests/assets/suc version

Reply:
true, I'll give this a try.

Reply:
done